content/en/docs/tasks/traffic-management/ingress/gateway-api/index.md

```diff
@@ -223,6 +223,68 @@ Note: only one address may be specified.
 
 * (Advanced) The generated Pod configuration can be configured by [Custom Injection Templates](/docs/setup/additional-setup/sidecar-injection/#custom-templates-experimental).
 
+#### Resource Attachment and Scaling
+
+{{< warning >}}
+Resource attachment is current experimental.
+{{< /warning >}}
+
+To customize the Gateway, resources can be *attached* to the `Gateway`.
+Since most APIs do not directly support attaching, they can instead be attached to the generated `Deployment` and `Service`.
+
+Both of these objects are guaranteed to be named the same as the `Gateway` and have a label `istio.io/gateway-name: GATEWAY_NAME`.
```

Comment:
```suggestion
Resources can be *attached* to a `Gateway` to customize it.
However, most Kubernetes resources do not currently support attaching directly to a `Gateway`, but they can be attached to the corresponding generated `Deployment` and `Service` instead.
This is easily done because both of these resources are generated with the same name as the `Gateway` and have a label `istio.io/gateway-name: <gateway name>`.
```